src/plugins/discover/public/application/components/discover_legacy.tsx

```diff
@@ -148,10 +163,46 @@ export function DiscoverLegacy({
     closed: isSidebarClosed,
   });
 
-  const mainSectionClassName = classNames({
-    'col-md-10': !isSidebarClosed,
-    'col-md-12': isSidebarClosed,
-  });
+  let flyout;
+
+  if (isFlyoutVisible) {
+    flyout = (
```

Comment:
I think it's not displayed, because the flyout variable isn't used here, just assigned. Btw, would be neat to have a separate flyout react component, to deduplicate code ... but could also be done later